session/session.go

```diff
@@ -2853,7 +2854,52 @@ func loadCollationParameter(se *session) (bool, error) {
 	return false, nil
 }
 
-var errResultIsEmpty = dbterror.ClassExecutor.NewStd(errno.ErrResultIsEmpty)
+var (
+	errResultIsEmpty = dbterror.ClassExecutor.NewStd(errno.ErrResultIsEmpty)
```

Comment:
```dbterror.ClassExecutor.NewStd(errno.ErrResultIsEmpty)``` has two place to define.

the other is at ```executor/errors.go:29```.

Reply:
well, I did not change anything here, just make it in the var block. If we want to unify the error definition, we can file another PR

Reply:
OK